### ISSUE TITLE
For commands that pass in the suggestion ID, escape non-letter/number characters

### ISSUE THREAD

**Issue by acollierr17** [suggestionsbot/suggestions-bot-js#83]
Based on the screenshot below, it can be problematic. I think this is in general will help going forward to alleviate any user-caused errors. I could also apply this to commands where the message ID is taken as an argument, but that might be a separate issue.
![image](https://user-images.githubusercontent.com/16583889/157584544-1e221bf4-2866-4906-83d5-8e757ff1b358.png)

**Comment by alaninnovates** [comment]
The regex `[^0-9a-z]` can be used to match any non-alphanumeric character. Then, the replace function can be used.

**Comment by acollierr17** [comment]
> The regex `[^0-9a-z]` can be used to match any non-alphanumeric character. Then, the replace function can be used.

Yup, that's exactly what I'll be doing. I actually have that regex pattern committed to memory for some reason :joy:
